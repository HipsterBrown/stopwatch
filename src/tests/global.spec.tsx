import renderer from 'react-test-renderer';
import React from 'react';
import GlobalStyles from '../global';

describe('global styles', () => {
  it('renders without correctly', () => {
    expect(renderer.create(<GlobalStyles />).toJSON()).toMatchSnapshot();
  });
});
