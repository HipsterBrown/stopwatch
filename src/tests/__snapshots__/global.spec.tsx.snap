// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`global styles renders without correctly 1`] = `null`;
